src/main/java/teammates/ui/webapi/Action.java

```diff
@@ -227,6 +247,8 @@ FeedbackSessionAttributes getNonNullFeedbackSession(String feedbackSessionName,
             if (studentAttributes == null) {
                 return Optional.empty();
             }
+            userInfoForUserWithNoAccount.put("regkey", key);
+            userInfoForUserWithNoAccount.put("email", studentAttributes.getEmail());
```

Comment:
Why not just keep the student object within the Action object itself?

Reply:
Understood. It is updated.